docroot/modules/custom/mass_metatag/tests/src/ExistingSite/CollectionsMetadataTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\Tests\mass_metatag\ExistingSite;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\paragraphs\Entity\Paragraph;
+use Drupal\taxonomy\Entity\Vocabulary;
+use Drupal\Tests\mass_metatag\Traits\TestContentTrait;
+
+/**
+ * Collections metadata tests.
+ */
+class CollectionsMetadataTest extends MetadataTestCase {
+
+  use TestContentTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getContent(): ContentEntityInterface {
```

Comment:
```suggestion
  public function createContent(): ContentEntityInterface {
```

Reply:
Renaming this causes issues, we should go deeper and rename the base method and everywhere it is used. So reverting to what it was